airflow/bin/cli.py

```diff
@@ -131,11 +131,22 @@ class CLIFactory:
         # list_dag_runs
         'no_backfill': Arg(
             ("--no_backfill",),
-            "filter all the backfill dagruns given the dag id", "store_true"),
```

Comment:
It's good practice to avoid whitespace/formatting only changes to code you don't touch.

Reply:
Done